src/utils.jl

```diff
@@ -40,6 +40,7 @@ function getNOctaves(n1,c::CWT{W,T, <:Dog, N}) where {W, T, N}
 end
 # choose the number of octaves so the smallest support is twice the qmf
 getNOctaves(n1,c::CWT{W,T, <:ContOrtho, N}) where {W, T, N} = log2(n1) - 2 - log2(length(qmf(c.waveType))) + c.extraOctaves
+getNOctaves(n1,c::CWT{W,T, Morse, N}) where {W, T, N} = log2((n1>>1+1)/(c.waveType.cf + 3)) + c.extraOctaves
```

Comment:
to test this (and getMinScaling), you'll need to add Morse() to the tests here https://github.com/UCD4IDS/ContinuousWavelets.jl/blob/79bf392248d7abde4ea076f843e63917231100d7/test/defaultProperties.jl#L4 and here https://github.com/UCD4IDS/ContinuousWavelets.jl/blob/79bf392248d7abde4ea076f843e63917231100d7/test/basicTypesAndNumber.jl#L7